### PR TITLE
Switching to .NETStandard

### PR DESCRIPTION
**Pull request by kekekeks**
- [x] Make everything to build
- [x] Update nuget packaging and test running
- [x] Make AppVeyor build working
- [x] See if solution can be built on travis (xbuild is disfuntional, so we need to compile the whole solution with msbuild)
- [ ] ???????
- [ ] Release

### CODE REVIEW

**Review by wieslawsoltes — APPROVED** [review]

File: tests/Avalonia.Benchmarks/packages.config [hunk 1]
```diff
@@ -1,7 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (grokys) [inline tests/Avalonia.Benchmarks/packages.config]: This file has been deleted from the filesystem but is still present in the project.

File: tests/Avalonia.Input.UnitTests/Avalonia.Input.UnitTests.csproj [hunk 2]
```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (grokys) [inline tests/Avalonia.Input.UnitTests/Avalonia.Input.UnitTests.csproj]: This project (and some other test project) doesn't seem to have been converted to the new project system, whereas other tests such as `Avalonia.Base.UnitTests` have been converted. Is there a reason for this?

File: src/Avalonia.Logging.Serilog/Avalonia.Logging.Serilog.csproj [hunk 3]
```diff
@@ -41,26 +29,14 @@
...
-      <HintPath>..\..\packages\Serilog.1.5.14\lib\portable-net45+win+wpa81+wp80+MonoAndroid10+MonoTouch10\Serilog.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Avalonia.Base\Avalonia.Base.csproj">
       <Project>{b09b78d8-9b26-48b0-9149-d64a2f120f3f}</Project>
```
Comment (grokys) [inline src/Avalonia.Logging.Serilog/Avalonia.Logging.Serilog.csproj]: As far as I can tell, project references using the new project system shouldn't have a `<Project>` element with a guid inside, and ncrunch is choking on this. I have gone through all the projects and removed these, do you want me to push the change to your branch?

**Review by grokys — CHANGES_REQUESTED** [review]

File: tests/Avalonia.Input.UnitTests/Avalonia.Input.UnitTests.csproj (hunk 2 above)
Comment (kekekeks) [inline tests/Avalonia.Input.UnitTests/Avalonia.Input.UnitTests.csproj]: No particular reason, I've stopped that conversion process after tests for `Avalonia.Controls` and `Avalonia.Layout` failed to run on .NET Core. We can address that later.

File: src/Avalonia.Logging.Serilog/Avalonia.Logging.Serilog.csproj (hunk 3 above)
Comment (kekekeks) [inline src/Avalonia.Logging.Serilog/Avalonia.Logging.Serilog.csproj]: It would be nice

**Pull request merged**